src/kube-runtime/deploy/rbac.yaml

```diff
@@ -18,17 +18,28 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: frameworkbarrier-account
+  name: runtime-account
   namespace: default
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   name: frameworkbarrier-role
 rules:
-- apiGroups: ["frameworkcontroller.microsoft.com"]
-  resources: ["frameworks"]
-  verbs: ["get", "watch", "list"]
+  - apiGroups: ["frameworkcontroller.microsoft.com"]
```

Comment:
why change the indents? better to follow k8s [example](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#referring-to-resources)

Reply:
Maybe formatter issue, will fix this

---
In reply to: [352482312](https://github.com/microsoft/pai/pull/3944#discussion_r352482312) [](ancestors = 352482312)